src/app/examples/card-example/card-example.component.tns.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../kirby/scss/native-imports';
+
+$bacon-height: 120;
+$bacon-width: 120;
+
+kirby-card-header {
+  color: var(--kirby-on-primary);
+  background-color: red;
+  padding-bottom: $bacon-width / 2;
+  margin-bottom: $bacon-width / 2;
```

Comment:
width should probably not used  for something related to a height (bottom)??

Reply:
Nej, jeg har fjernet alt styling i filen, det blev ikke brugt